src/settings.cpp

```diff
@@ -445,6 +445,19 @@ vstrCentralServerAddress[0] = GetIniSetting ( IniXMLDocument, "client", "central
         eCentralServerAddressType = AT_DEFAULT;
     }
 
+    // custom directory server index
+    if ( GetNumericIniSet ( IniXMLDocument, "client", "customdirectoryindex", 0, MAX_NUM_SERVER_ADDR_ITEMS, iValue ) )
+    {
+        iCustomDirectoryIndex = iValue ;
```

Comment:
No space needed after iValue

Reply:
yes local clang-format was invented for people like me